### PR TITLE
Add plus operator idea from concat

### PR DESCRIPTION
**Pull request by hoangchungk53qx1**
## Status

**READY**

## Breaking Changes

NO

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

## Requirements

- [ ✅ ] Dump api
- [ ✅ ] Update README
- [ ✅ ] Update CHANGELOG
- [ ✅ ] Add tests
- [ ✅ ] Add docs

### CODE REVIEW

**Comment by hoc081098** [comment]
🙏

File: CHANGELOG.md [hunk 1]
```diff
@@ -29,6 +29,7 @@
 
 - Add `Flow.zipWithNext()` operator, it is an alias to `Flow.pairwise()` operator.
 - Add `Flow.zipWithNext(transform)` operator, it is an alias to `Flow.pairwise(transform)` operator.
+- Add `Flow.plus` operator.
```
Comment (hoc081098) [inline CHANGELOG.md:32]: Move to `unreleased > Added` section

```suggestion
- Add `Flow.plus` operator, it is an alias to `concatWith` operator.
```

File: README.md [hunk 2]
```diff
@@ -1303,6 +1304,28 @@ withLatestFrom: (2, 3)
...
+
+```none
+plus: 1
+plus: 2
+plus: 3
+plus: 4
+plus: 5
+plus: 6
+```
+
```
Comment (hoc081098) [inline README.md:1328]: Unify with `concatWith`.
Similar to `pairwise() / zipWithNext()`, ...

File: README.md [hunk 3]
```diff
@@ -121,6 +121,7 @@ dependencies {
   - [`amb`](#race--amb)
   - [`range`](#range)
   - [`timer`](#timer)
+  - [`plus`](#plus)
```
Comment (hoc081098) [inline README.md:124]: Unify with `concatWith`.
Similar to `pairwise() / zipWithNext()`, ...

File: .idea/vcs.xml [hunk 4]
```diff
@@ -6,7 +6,4 @@
       <inspection_tool class="CommitNamingConvention" enabled="true" level="WARNING" enabled_by_default="true" />
     </profile>
   </component>
-  <component name="VcsDirectoryMappings">
-    <mapping directory="$PROJECT_DIR$" vcs="Git" />
-  </component>
-</project>
+</project>
```
Comment (hoc081098) [inline .idea/vcs.xml:9]:
```suggestion
  <component name="VcsDirectoryMappings">
    <mapping directory="$PROJECT_DIR$" vcs="Git" />
  </component>
</project>
```

File: README.md [hunk 5]
```diff
@@ -1303,6 +1319,9 @@ withLatestFrom: (2, 3)
 withLatestFrom: (3, 4)
 withLatestFrom: (4, 6)
 ```
+#### plus
+
+
```
Comment (hoc081098) [inline README.md:1324]:
```suggestion
```

File: .idea/vcs.xml [hunk 6]
```diff
@@ -9,4 +9,4 @@
   <component name="VcsDirectoryMappings">
     <mapping directory="$PROJECT_DIR$" vcs="Git" />
   </component>
-</project>
+</project>
```
Comment (hoc081098) [inline .idea/vcs.xml:12]:
```suggestion
</project>

```

File: README.md
Comment (hoc081098) [inline README.md:1]: `plus` should be the same section with `concatWith` (operator), not `concat` (create)

File: src/commonTest/kotlin/com/hoc081098/flowext/ConcatTest.kt
Comment (hoc081098) [inline src/commonTest/kotlin/com/hoc081098/flowext/ConcatTest.kt:1]: `plus` should be the same section with `concatWith` (operator), not `concat` (create)

File: src/commonMain/kotlin/com/hoc081098/flowext/concat.kt [hunk 7]
```diff
@@ -267,3 +267,19 @@ public fun <T> Flow<T>.startWith(others: Sequence<T>): Flow<T> = concat(others.a
...
+ *
+ * Example:
+ *
+ * ``` kotlin
+ *  val flow1 = flowOf(1, 2, 3)
+ *  val flow2 = flowOf(4, 5, 6)
+ *  val plusFlow = flow1 + flow2 //1, 2, 3, 4, 5, 6
+ * ```
+ */
+
```
Comment (hoc081098) [inline src/commonMain/kotlin/com/hoc081098/flowext/concat.kt:284]:
```suggestion

/**
 * This function is an alias to [concatWith] operator.
 *
 * Combines two [Flow]s of the same base type [T] into a single [Flow] by concatenating their elements.
 *
 * @param other The [Flow] to concatenate with the current [Flow].
 * @return A new [Flow] containing the concatenated elements of both the current and the [other] [Flow]s.
 * @see concatWith
 *
 * Example:
 * ``` kotlin
 *  val flow1 = flowOf(1, 2, 3)
 *  val flow2 = flowOf(4, 5, 6)
 *  val result = flow1 + flow2 // produces the following emissions 1, 2, 3, 4, 5, 6
 * ```
 */
```

**Comment by hoangchungk53qx1** [comment]
🙏 💯 🔢 🥇

**Review by hoc081098 — APPROVED** [review]
🙏🎆

**Pull request merged**